Bug: test_vxlan_decap_ttl.py fail on dualtor

### Is it platform specific

generic

### Importance or Severity

Critical

### Description of the bug

current test test_vxlan_decap_ttl.py cannot will fail on dualtor topo, is it in expectation? if so we may need to add it in skip condition.

### Steps to Reproduce

run test_vxlan_decap_ttl.py on dualtor topo

### Actual Behavior and Expected Behavior

we expect this test to be skipped or succeed on dualtor topo

### Relevant log output

```shell
AssertionError: Did not receive expected packet on any of ports [64, 65] for device 0.
========== EXPECTED ==========
Mask:

packet status: OK
packet:
0000  9C 05 91 E7 49 00 46 15 33 EA B0 00 08 00 45 00  ....I.F.3.....E.
0010  00 56 00 01 00 00 01 11 EE B4 C0 A8 00 01 0A 00  .V..............
0020  00 39 04 D2 00 50 00 42 A7 7F 74 65 73 74 73 2E  .9...P.B..tests.
0030  76 78 6C 61 6E 2E 74 65 73 74 5F 76 78 6C 61 6E  vxlan.test_vxlan
0040  5F 64 65 63 61 70 5F 74 74 6C 20 74 65 73 74 73  _decap_ttl tests
0050  2E 76 78 6C 61 6E 2E 74 65 73 74 5F 76 78 6C 61  .vxlan.test_vxla
0060  6E 5F 64 65                                      n_de

packet's mask:
0000  00 00 00 00 00 00 00 00 00 00 00 00 FF FF F0 00  ................
0010  FF FF 00 00 1F FF FF FF 00 00 FF FF FF FF FF FF  ................
0020  FF FF FF FF FF FF FF FF 00 00 FF FF FF FF FF FF  ................
0030  FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF  ................
0040  FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF  ................
0050  FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF  ................
0060  FF FF FF FF                                      ....

========== RECEIVED ==========
283 total packets. Displaying most recent 3 packets:
------------------------------
0000  0E BD BD E2 F5 B4 9C 05 91 E7 49 00 08 00 45 C0  ..........I...E.
0010  00 47 A9 9F 40 00 01 06 BA D5 0A 00 00 3E 0A 00  .G..@........>..
0020  00 3F B6 9C 00 B3 B6 2B EE 8F A7 08 99 AE 80 18  .?.....+........
0030  00 8A EB BD 00 00 01 01 08 0A 5E 71 AE 35 06 E0  ..........^q.5..
0040  C1 81 FF FF FF FF FF FF FF FF FF FF FF FF FF FF  ................
0050  FF FF 00 13 04                                   .....
------------------------------
0000  0A 8C 83 0D B9 E7 9C 05 91 30 F8 00 08 00 45 C0  .........0....E.
0010  00 34 E6 2C 40 00 01 06 7C 67 0A 00 01 38 0A 00  .4.,@...|g...8..
0020  01 39 A3 0A 00 B3 19 3C FB 3F 1D 1B 6F 45 80 10  .9.....<.?..oE..
0030  00 8A 0D 9E 00 00 01 01 08 0A 3A 7D 3F 44 A9 F8  ..........:}?D..
0040  E9 D0                                            ..
------------------------------
0000  2A 9D C5 A7 5F B7 9C 05 91 E7 49 00 86 DD 6C 01  *..._.....I...l.
0010  65 13 00 20 06 01 FC 00 00 00 00 00 00 00 00 00  e.. ............
0020  00 00 00 00 00 75 FC 00 00 00 00 00 00 00 00 00  .....u..........
0030  00 00 00 00 00 76 00 B3 A4 6F E8 22 6B C6 A0 0F  .....v...o."k...
0040  2C 0F 80 10 03 10 31 26 00 00 01 01 08 0A 2B DF  ,.....1&......+.
0050  AC 7D 7C 0F 30 04                                .}|.0.
==============================
duthost = 
tbinfo = {'comment': '4700 Active-Active Dual-TOR testbed', 'conf-name': 'sonic-dual-tor-leopard-dualtor-aa-64-breakout', 'duts': ['r-leopard-70', 'r-leopard-72'], 'duts_map': {'r-leopard-70': 0, 'r-leopard-72': 1}, ...}
ptfadapter = 
create_vnet = 'Vnet-0', outer_ip_version = 'v4', inner_ip_version = 'v4'

    def test_vxlan_decap_ttl(duthost, tbinfo, ptfadapter, create_vnet, outer_ip_version, inner_ip_version):  # noqa F811
        """
            In this test, the DUT acts as a VNET endpoint and decapulates VxLAN packets sent to it that match
            the VNI of the VNET configured on it.
            The test verifies that TTL/Hop limit of the egress packet is set correctly when using the pipe model
            (i.e., to the TTL/Hop limit of the inner ingress packet minus 1).
        """
        minigraph_facts = duthost.get_extended_minigraph_facts(tbinfo)
        router_mac = duthost.facts["router_mac"]
        vnet_endpoint = ecmp_utils.get_dut_loopback_address(duthost, minigraph_facts, outer_ip_version)
        ptf_indices = minigraph_facts["minigraph_ptf_indices"]
        ingress_port = select_ingress_port(duthost)
        inner_dst_ip, egress_ports = select_egress_ip_and_ports(duthost, minigraph_facts,
                                                                inner_ip_version, exclude_ports=[ingress_port])
        egress_port_indices = [ptf_indices[port] for port in egress_ports]
        ptf_src_mac = ptfadapter.dataplane.get_mac(0, ptf_indices[ingress_port])
    
        inner_pkt = get_inner_packet(dst_mac=router_mac, src_mac=ptf_src_mac, ip_version=inner_ip_version,
                                     dst_ip=inner_dst_ip, ttl=2)
        outer_pkt = get_outer_packet(eth_dst=router_mac, eth_src=ptf_src_mac, ip_version=outer_ip_version,
                                     ip_dst=vnet_endpoint, inner_pkt=inner_pkt)
    
        exp_pkt_mask = get_expected_packet_mask(inner_pkt, inner_ip_version)
        ptfadapter.dataplane.flush()
        testutils.send(ptfadapter, ptf_indices[ingress_port], outer_pkt)
>       _, received_pkt = testutils.verify_packet_any_port(ptfadapter, exp_pkt_mask, egress_port_indices)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

create_vnet = 'Vnet-0'
duthost    = 
egress_port_indices = [64, 65]
egress_ports = ['Ethernet96', 'Ethernet104']
exp_pkt_mask = 
ingress_port = 'Ethernet0'
inner_dst_ip = '10.0.0.57'
inner_ip_version = 'v4'
inner_pkt  = >>>
minigraph_facts = {'deployment_id': '1', 'dhcp_servers': ['192.0.0.1', '192.0.0.2', '192.0.0.3', '192.0.0.4'], 'dhcpv6_servers': ['fc02:...routes': ['10.75.0.0/16', '10.213.0.0/16', '10.215.0.0/16', '10.9.0.0/16', '10.212.0.0/16', '10.245.0.0/16', ...], ...}
outer_ip_version = 'v4'
outer_pkt  = >>>>>>>
ptf_indices = {'Ethernet0': 0, 'Ethernet104': 65, 'Ethernet112': 68, 'Ethernet12': 3, ...}
ptf_src_mac = b'46:15:33:ea:b0:00'
ptfadapter = 
router_mac = '9c:05:91:e7:49:00'
tbinfo     = {'comment': '4700 Active-Active Dual-TOR testbed', 'conf-name': 'sonic-dual-tor-leopard-dualtor-aa-64-breakout', 'duts': ['r-leopard-70', 'r-leopard-72'], 'duts_map': {'r-leopard-70': 0, 'r-leopard-72': 1}, ...}
vnet_endpoint = '10.1.0.32'

vxlan/test_vxlan_decap_ttl.py:246: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/opt/venv/lib/python3.12/site-packages/ptf/testutils.py:3441: in verify_packet_any_port
    test.fail(
        device_number = 0
        n_timeout  = 0.1
        pkt        = 
        ports      = [64, 65]
        result     = PollFailure(device=None, port=None, packet=None, time=None)
        test       = 
        timeout    = 2
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = 
msg = 'Did not receive expected packet on any of ports [64, 65] for device 0.\n========== EXPECTED ==========\nMask:\n\npack... DF  ,.....1&......+.\n0050  AC 7D 7C 0F 30 04                                .}|.0.\n==============================\n'

    def fail(self, msg=None):
        """Fail immediately, with the given message."""
>       raise self.failureException(msg)
E       AssertionError: Did not receive expected packet on any of ports [64, 65] for device 0.
E       ========== EXPECTED ==========
E       Mask:
E       
E       packet status: OK
E       packet:
E       0000  9C 05 91 E7 49 00 46 15 33 EA B0 00 08 00 45 00  ....I.F.3.....E.
E       0010  00 56 00 01 00 00 01 11 EE B4 C0 A8 00 01 0A 00  .V..............
E       0020  00 39 04 D2 00 50 00 42 A7 7F 74 65 73 74 73 2E  .9...P.B..tests.
E       0030  76 78 6C 61 6E 2E 74 65 73 74 5F 76 78 6C 61 6E  vxlan.test_vxlan
E       0040  5F 64 65 63 61 70 5F 74 74 6C 20 74 65 73 74 73  _decap_ttl tests
E       0050  2E 76 78 6C 61 6E 2E 74 65 73 74 5F 76 78 6C 61  .vxlan.test_vxla
E       0060  6E 5F 64 65                                      n_de
E       
E       packet's mask:
E       0000  00 00 00 00 00 00 00 00 00 00 00 00 FF FF F0 00  ................
E       0010  FF FF 00 00 1F FF FF FF 00 00 FF FF FF FF FF FF  ................
E       0020  FF FF FF FF FF FF FF FF 00 00 FF FF FF FF FF FF  ................
E       0030  FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF  ................
E       0040  FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF  ................
E       0050  FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF  ................
E       0060  FF FF FF FF                                      ....
E       
E       ========== RECEIVED ==========
E       283 total packets. Displaying most recent 3 packets:
E       ------------------------------
E       0000  0E BD BD E2 F5 B4 9C 05 91 E7 49 00 08 00 45 C0  ..........I...E.
E       0010  00 47 A9 9F 40 00 01 06 BA D5 0A 00 00 3E 0A 00  .G..@........>..
E       0020  00 3F B6 9C 00 B3 B6 2B EE 8F A7 08 99 AE 80 18  .?.....+........
E       0030  00 8A EB BD 00 00 01 01 08 0A 5E 71 AE 35 06 E0  ..........^q.5..
E       0040  C1 81 FF FF FF FF FF FF FF FF FF FF FF FF FF FF  ................
E       0050  FF FF 00 13 04                                   .....
E       ------------------------------
E       0000  0A 8C 83 0D B9 E7 9C 05 91 30 F8 00 08 00 45 C0  .........0....E.
E       0010  00 34 E6 2C 40 00 01 06 7C 67 0A 00 01 38 0A 00  .4.,@...|g...8..
E       0020  01 39 A3 0A 00 B3 19 3C FB 3F 1D 1B 6F 45 80 10  .9.....<.?..oE..
E       0030  00 8A 0D 9E 00 00 01 01 08 0A 3A 7D 3F 44 A9 F8  ..........:}?D..
E       0040  E9 D0                                            ..
E       ------------------------------
E       0000  2A 9D C5 A7 5F B7 9C 05 91 E7 49 00 86 DD 6C 01  *..._.....I...l.
E       0010  65 13 00 20 06 01 FC 00 00 00 00 00 00 00 00 00  e.. ............
E       0020  00 00 00 00 00 75 FC 00 00 00 00 00 00 00 00 00  .....u..........
E       0030  00 00 00 00 00 76 00 B3 A4 6F E8 22 6B C6 A0 0F  .....v...o."k...
E       0040  2C 0F 80 10 03 10 31 26 00 00 01 01 08 0A 2B DF  ,.....1&......+.
E       0050  AC 7D 7C 0F 30 04                                .}|.0.
E       ==============================

msg        = 'Did not receive expected packet on any of ports [64, 65] for device 0.\n========== EXPECTED ==========\nMask:\n\npack... DF  ,.....1&......+.\n0050  AC 7D 7C 0F 30 04                                .}|.0.\n==============================\n'
self       = 

/usr/lib/python3.12/unittest/case.py:715: AssertionError
```

### Output of `show version`

```shell

```

### Attach files (if any)

_No response_

--- a/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml
+++ b/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml
@@ -5983,9 +5983,12 @@ vxlan/test_vxlan_decap.py:
 
 vxlan/test_vxlan_decap_ttl.py:
   skip:
-    reason: "VxLAN tunnel TTL decap mode test is not supported on multi-ASIC platform. Also this test can only run and currently passes only on some platforms."
+    reason: "VxLAN tunnel TTL decap mode test is not supported on multi-ASIC platforms and dualtor topologies. Also this test can only run and currently passes only on some platforms."
+    conditions_logical_operator: OR
     conditions:
-      - "(is_multi_asic == True) or (platform not in ['x86_64-8102_64h_o-r0', 'x86_64-8101_32fh_o-r0', 'x86_64-mlnx_msn4600c-r0', 'x86_64-mlnx_msn2700-r0', 'x86_64-mlnx_msn2700a1-r0', 'x86_64-mlnx_msn4700-r0', 'x86_64-nvidia_sn4280-r0', 'x86_64-8102_28fh_dpu_o-r0'])"
+      - "is_multi_asic == True"
+      - "platform not in ['x86_64-8102_64h_o-r0', 'x86_64-8101_32fh_o-r0', 'x86_64-mlnx_msn4600c-r0', 'x86_64-mlnx_msn2700-r0', 'x86_64-mlnx_msn2700a1-r0', 'x86_64-mlnx_msn4700-r0', 'x86_64-nvidia_sn4280-r0', 'x86_64-8102_28fh_dpu_o-r0']"
+      - "'dualtor' in topo_name"
 
 vxlan/test_vxlan_ecmp.py:
   skip:

